packages/vuetify/src/components/VSelect/VSelect.sass

```diff
@@ -7,6 +7,9 @@
   .v-select__selection--disabled
     color: map-deep-get($material, 'text', 'disabled')
 
+  .v-select__selection--dense
+    margin: 0px 4px 0px
```

Comment:
is this something that we could get from sass vars?

Reply:
Do you mean adding a `$select-selections-margin-dense: 0 4px 0 !default;` to 
`vuetify/packages/vuetify/src/components/VSelect/_variables.scss`, or do you mean using a variable that already exists like `$spacer` from` /styles/settings/_variables.scss`?

Reply:
I meant using some existing variable from `(VSelect/VTextField/VInput)/_variables.scss` that makes displaying autocomplete properly, but can't find anything that matches. So adding `$select-selections-margin-dense: 0 4px 0 !default` to select variables should be fine

Also please add `chips` and `small-chips` variants to the playground - does your change work well with them?

Reply:
https://codepen.io/amoliski/pen/NWqjdyR
Before the change, the empty and the filled select have different heights, but the empty/filled/autocomplete small-chips all have the same (slightly taller) height.

With my fix:
![With fix](https://user-images.githubusercontent.com/880466/75478288-4efc1880-5963-11ea-8ee0-84a453cda5c4.png)

Without my fix:
![Without fix](https://user-images.githubusercontent.com/880466/75478290-4f94af00-5963-11ea-92df-22d10b453a7a.png)

I _think_ this is the desired behavior- with chips is slightly taller than without in both dense and standard mode to account for the chips padding. At the very least, it matches the autocomplete behavior.

Reply:
Actually looking at it again, the real solution might be to ADD padding to the empty dense select instead of removing it from the non-empty dense select. With the added padding, it will line up with the dense autcomplete, and without the padding, it might be _too_ dense.

With Fix: Both dense selects are shorter than the dense autocompletes
![With fix 2](https://user-images.githubusercontent.com/880466/75480339-b7002e00-5966-11ea-98bd-c5e97819ca24.png)

Without fix, only the empty select is shorter than the autocomplete
![Without Fix 2](https://user-images.githubusercontent.com/880466/75480341-b798c480-5966-11ea-8c97-24fe0cfd7b59.png)

Reply:
It looks like all it actually takes to fix this is to add `:not(.v-select)` to line 89 of `VTextField.sass`

```
&.v-input--dense
    padding-top: 0

    &:not(.v-text-field--outlined):not(.v-text-field--single-line):not(.v-select) //Here
      input
        padding: $text-field-dense-padding

    &[type=text]::-ms-clear
      display: none
```

With that dense text field padding removed, everything lines up as it should:
![actual fix](https://user-images.githubusercontent.com/880466/75481040-0430cf80-5968-11ea-85b9-70f648cf554c.png)